python/tests/test_translate.py

```diff
@@ -1,27 +1,55 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
 
+import string
+
+import pytest
+import pandas as pd
 import nvstrings
 
-#
-from librmm_cffi import librmm as rmm
-from librmm_cffi import librmm_config as rmm_cfg
-rmm_cfg.use_pool_allocator = True 
-rmm.initialize()
+from utils import assert_eq
 
-strs = nvstrings.to_device(["hello","there","world","accéntéd",None,""])
-print(strs)
-print(".translate():",strs.translate([]))
-print(".translate([[e,a]]):",strs.translate([['e','a']]))
-print(".translate([[e,é]]):",strs.translate([['e','é']]))
-print(".translate([[é,e],[o,None]]):",strs.translate([['é','e'],['o',None]]))
 
-print(".translate(maketrans(e,a):",strs.translate(str.maketrans('e','a')))
-print(".translate(maketrans(elh,ELH):",strs.translate(str.maketrans('elh','ELH')))
+@pytest.mark.parametrize('table', [
```

Comment:
What is happening here? Is Pandas required to verify this translate?

Reply:
Pandas is not required, but in some areas may be desirable for consistency with the `series.str` API. No individual test inherently needs to be comparing with pandas. We could compare this to the Python translate API, which would also require an ordinal mapping

Reply:
You should be able to use the same translate-table for both nvstrings and pandas:
```
import pandas as pd
import nvstrings
ttbl = str.maketrans('e','a')
ss = ['hello','there','world','accéntéd',None,'']
ps = pd.Series(ss)
ds = nvstrings.to_device(ss)
print(ps.str.translate(ttbl))
print(ds.translate(ttbl))
```
```
0       hallo
1       thara
2       world
3    accéntéd
4        None
5            
dtype: object

['hallo', 'thara', 'world', 'accéntéd', None, '']
```
If that helps.

Reply:
This does help, thanks! The reason I have them separate is that the pandas method exclusively accepts an ordinal:character mapping dictionary (such as from the translate table), but the nvstrings method does not. It can accept the mapping as either a list of a dictionary. In the ordinal test, its doing what you've described with the `str.maketrans('elh', 'ELH'),` parameter to the pytest. Let me know if you think we should change/combine them!

Reply:
@davidwendt given our Slack conversation, are we good to consider this resolved for now with the plan to transition away in the future?

Reply:
Yes